ui/src/pages/common.ts

```diff
@@ -54,3 +54,15 @@ export function damlSetValues<T>(damlSet: DamlSet<T>): T[] {
   }
   return r;
 }
+
+export function partitionArray<T>(isValid: (elem: T) => boolean, array?: T[]): [T[], T[]] {
```

Comment:
This looks identical to `_.partition` in the `lodash` library.

From the lodash docs at see https://lodash.com/docs/4.17.15#partition :

> Creates an array of elements split into two groups, the first of which contains elements predicate returns truthy for, the second of which contains elements predicate returns falsey for. The predicate is invoked with one argument: (value).